### DISCUSSION TITLE
Setting-Up my Backend and DB

### DISCUSSION THREAD

**Discussion by AgusMaris** [Nozbe/WatermelonDB#1086] [Q&A] [answered]
Hi people, im planning to use WatermelonDB as my offline-first choice for an App. I want to build a backend in .NET Core to sync it with my MSSQL-DB, what should i store additionally in my MSSQL DB in order to implement the sync feature? I mean, should i add columns called "created_at" "updated_at" and "deleted_at" with timestamp values in each table in order to query them?

Thanks

**Comment by radex** [comment, marked as answer]
Read this from top to bottom, it should tell you all you need to know: https://github.com/Nozbe/WatermelonDB/blob/master/docs-master/Advanced/Sync.md

**Comment by hlovdal** [comment]
That document seems to be split up and moved to https://github.com/Nozbe/WatermelonDB/tree/master/docs-website/docs/docs/Sync.
